linetools/spectralline.py

```diff
@@ -407,7 +408,7 @@ def fill_data(self, trans, linelist=None, closest=False):
 
         # Update
         self.wrest = self.data['wrest']
-        self.trans = self.data['name']
+        self.name = self.data['name']
```

Comment:
Either trans or name has to be only something like:

SiII 1526

X

> On Jan 17, 2016, at 10:06 AM, Nicolas Tejos <notifications@github.com> wrote:
> 
> In linetools/spectralline.py <https://github.com/linetools/linetools/pull/150#discussion_r49949807>:
> 
> > @@ -407,7 +408,7 @@ def fill_data(self, trans, linelist=None, closest=False):
> >  
> >          # Update
> >          self.wrest = self.data['wrest']
> > -        self.trans = self.data['name']
> > +        self.name = self.data['name']
> I'm wondering now, should the name of the AbsLine contain redshift information? (Default could be z=0)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/linetools/linetools/pull/150/files#r49949807>.
> 
> !DSPAM:826,569bd8116408191712217!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)